CONTRIBUTING.md

```diff
@@ -0,0 +1,28 @@
+## Contributing to saadaal-flood-forecaster
+
+Thank you for contributing to Saadaal Flood Forecaster.
+
+To follow the project, please take a look at our [backlog](https://github.com/orgs/saadaal-dev/projects/1). You can take tasks from there or propose more ideas while tracking them.
+
+For reporting a bug, you can open an issue in our [backlog](https://github.com/orgs/saadaal-dev/projects/1).
```

Comment:
Might be easier to open a bug directly from https://github.com/saadaal-dev/saadaal-flood-forecaster/issues; BTW just noticed that all toDo tassks are marked as bugs :)